packages/rocketchat-lib/package.js

```diff
@@ -24,7 +24,7 @@ Package.onUse(function(api) {
 	api.use('ddp-rate-limiter');
 	api.use('underscore');
 	api.use('mongo');
-	api.use('underscorestring:underscore.string');
+	api.use('underscorestring:underscore.string@3.3.4');
```

Comment:
We already had `underscorestring:underscore.string@3.3.4` defined on `.meteor/versions` that should have done that.